docs/src/pages/components/data-grid/columns/columns.md

```diff
@@ -50,6 +50,14 @@ To change the width of a column, use the `width` property available in `GridColD
 
 {{"demo": "pages/components/data-grid/columns/ColumnWidthGrid.js", "bg": "inline"}}
 
+### Minimum width
+
+By default, the columns have a minimum width of 50 pixels.
```

Comment:
We have moved away from using "pixels" in the core because it could be confused with the screen pixels, here we cover CSS pixels, so px seems more appropriate. Would it make more sense? It's almost the same so not something major.

```suggestion
By default, the columns have a minimum width of 50px.
```